packages/jaeger-ui/src/components/DeepDependencies/Graph/index.css

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) 2019 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.DdgArrow.is-pathHovered {
+  fill: #08979c;
+  stroke: none;
+}
+
+.DdgEdge.is-pathHovered {
```

Comment:
updated so that color does not change, only opacity. this has a slight issue such that when fully zoomed in, the appearances of matches does not change. when only matches are in view, then there doesn't seem to be a response.
I want to keep stroke color reserved for other decorations so I suppose that is a necessary trade off for now.